typings/model.d.ts

```diff
@@ -92,8 +119,8 @@ declare interface FieldGoal {
 }
 
 declare interface ThreePoint {
```

Comment:
seeing this stuff, this could be ideal for virtuals instead of creating its own model and interface. work it out for virtual first. delete this after.

Reply:
noted, thought about that too, will remove once everything works well without using those.